### PR TITLE
Gradle plugin docs

### PR DESCRIPTION
**Pull request by Isammoc**
Migrate the gradle plugin documentation.

cc @eshepelyuk @slandelle

### CODE REVIEW

**Review by eshepelyuk — APPROVED** [review]

File: src/sphinx/extensions/gradle_plugin.rst [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+         <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+         <immediateFlush>false</immediateFlush>
+       </encoder>
+     </appender>
+     <root level="${logLevel}"> <!--1-->
+       <appender-ref ref="CONSOLE"/>
+     </root>
+   </configuration>
+
+1. ``logLevel`` is configured via plugin extension, ``WARN`` by default.
```
Comment (slandelle) [inline src/sphinx/extensions/gradle_plugin.rst:204]: Missing mention of the effect of `logHttp`.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: src/sphinx/extensions/gradle_plugin.rst [hunk 2]
```diff
@@ -0,0 +1,358 @@
+.. _gradle-plugin:
+
+#############
+Gradle plugin
+#############
+
+This Gradle plugin integrates Gatling with Gradle, allowing to use Gatling as a testing framework.
+
```
Comment (slandelle) [inline src/sphinx/extensions/gradle_plugin.rst:8]: Missing mention of the original contributors.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: src/sphinx/extensions/gradle_plugin.rst (hunk 1 above)
Comment (Isammoc) [reply src/sphinx/extensions/gradle_plugin.rst:204]: Wanted: it is not in the default configuration.

But you're right, I added another example.

File: src/sphinx/extensions/gradle_plugin.rst (hunk 2 above)
Comment (Isammoc) [reply src/sphinx/extensions/gradle_plugin.rst:8]: Done

File: src/sphinx/extensions/gradle_plugin.rst [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+   These options ensure that ``4.0.51.Final`` will be used only for ``gatling``
+   configurations, leaving other dependencies unchanged.
+
+Sources
+=======
+
+If you're interested in contributing, you can find the `io.gatling.gradle plugin sources <https://github.com/gatling/gatling-gradle-plugin>`_ on Github.
+
+Acknowledgements
+================
```
Comment (slandelle) [inline src/sphinx/extensions/gradle_plugin.rst:382]: Please move up, just like it's being done for https://gatling.io/docs/current/http/websocket/ and https://gatling.io/docs/current/jms/.

**Review by slandelle — CHANGES_REQUESTED** [review]

**Review by slandelle — APPROVED** [review]

**Pull request closed without merging**